src/components/TransactionsList.js

```diff
@@ -28,7 +29,12 @@ const TransactionsList = () => {
           </th>
         </tr>
 
-        {"... your code here..."}
+        {props.transactions.map(function(t, index) {
```

Comment:
might be able to do this using `.filter` and then `.map` or even reduce - may be more readable